src/cmds/tail.c

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file
+ * @brief head-util
+ *
+ * @date 07.09.18
+ * @author Filipp Chubukov
+ */
+
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <ctype.h>
+
+#define SIZE 100
+
+static void read_last_n_lines(FILE *fp, int n){
+	  int lines_checked = 0;  
```

Comment:
Please use Code-style use only tabs in a start line

Reply:
Yeah, I am sorry. Will correct it and take special care of it from next time.